packages/serverpod/lib/src/server/session.dart

```diff
@@ -307,24 +321,21 @@ class MethodCallSession extends Session {
 /// associated with the current connection and provides easy access to the
 /// database.
 class MethodStreamSession extends Session {
-  /// The name of the method that is being called.
-  final String methodName;
-
-  /// The name of the endpoint that is being called.
-  final String endpointName;
-
   /// The connection id that uniquely identifies the stream.
   final UuidValue connectionId;
 
+  @override
+  String get methodName => super.methodName!;
```

Comment:
Suggestion: We should probably use the same funky override here as for the `MethodCallSession` to get rid of the scary `!`.